src/GADM.jl

```diff
@@ -122,4 +123,24 @@ function get(country, subregions...)
     throw(ArgumentError("feature not found"))
 end
 
+"""
+    coordinates(country, levels...)  
+
+Returns a deep array of Float64 coordinates for the requested region.  
+Input: ISO 3166 Alpha 3 Country Code, and further full official names of subdivisions  
+
+## Examples  
+  
+```julia
+coordinates("IND") # Returns array of coordinates of India
+coordinates("IND", "Uttar Pradesh") # Returns array of state Uttar Pradesh
+coordinates("IND", "Uttar Pradesh", "Lucknow") # Returns array of coordinates of district Lucknow
+```
```

Comment:
Can we have a simple example with actual numbers? For example, copy/paste the first coordinates of the query with VAT.